Packs/Cyberint/Integrations/Cyberint/Cyberint.py

```diff
@@ -504,20 +540,41 @@ def fetch_incidents(client: Client, last_run: Dict[str, int],
                         attachments.append(tmp_attachment)
 
         alert["attachments"] = attachments
-        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
-        if alert_csv_id:
-            extracted_csv_data = extract_data_from_csv_stream(client, alert_id,  # type: ignore
-                                                              alert_csv_id)
-            alert['alert_data']['csv'] = extracted_csv_data
+        alert_data = dict_safe_get(alert, ['alert_data', 'csv'])
+        incident_csv_records = dict_safe_get(alert, ['alert_data', 'csv', 'content']) or []
 
         incident = {
             'name': f'Cyberint alert {alert_id}: {alert_title}',
             'occurred': datetime.strftime(alert_created_time, DATE_FORMAT),
             'rawJSON': json.dumps(alert),
             'severity': SEVERITIES.get(alert.get('severity', 'low'), 1),
-            'attachment': incident_attachments
+            'attachment': incident_attachments,
         }
-        incidents.append(incident)
+
+        alert_csv_id = alert.get('alert_data', {}).get('csv', {}).get('id', '')
+        if alert_csv_id:
+            extracted_csv_data = extract_data_from_csv_stream(
+                client,
+                alert_id,  # type: ignore
+                alert_csv_id)
+            alert['alert_data']['csv'] = extracted_csv_data
+
+        if duplicate_alert:
+            if incident_csv_records:
+                for index, incident_csv_record in enumerate(incident_csv_records):
+                    alert_data.update({'content': incident_csv_record})
+                    alert.update({'attachments': alert_data})
+                    incident.update({
+                        'name': f'Cyberint alert {alert_id} ({index+1}): {alert_title}',
+                        'rawJSON': json.dumps(alert)
+                    })
+                    incidents.append(copy.deepcopy(incident))
+            else:
+                incidents.append(incident)
+        else:
+
+            incident.update({'rawJSON': json.dumps(alert)})
+            incidents.append(incident)

```

Comment:
```suggestion
        if duplicate_alert and incident_csv_records:
                for index, incident_csv_record in enumerate(incident_csv_records):
                    alert_data.update({'content': incident_csv_record})
                    alert.update({'attachments': alert_data})
                    incident.update({
                        'name': f'Cyberint alert {alert_id} ({index+1}): {alert_title}',
                        'rawJSON': json.dumps(alert)
                    })
                    incidents.append(copy.deepcopy(incident))
        else:
            incidents.append(incident)

```
- I don't see any reason for separating between duplicate_alert as False and incident_csv_records as False. In both cases we want to just add a single incident. 
- Is there a specific reason that we need to use incident.update({'rawJSON': json.dumps(alert)}) in the else case? It seems that it already defined when you defined the incident.